server/src/controllers/user.ts

```diff
@@ -0,0 +1,19 @@
+import { Request, Response } from "express";
+import UserService from "../services/users";
+
+async function findUser(request: Request, response: Response): Promise<void> {
+  try {
+    const auth0Id = request.params.id;
```

Comment:
what is .params.id?

Reply:
so in routes/user.ts, i'm using `/:id`
the use case is, for example:
localhost:3001/user/123

123 replaces the template for :id
to retrieve the id from the url, I just do `request.params.id`
so i can use it to look for a user